submissions/DmitryHniezdilov/task-js-dom/js/main.js

```diff
@@ -0,0 +1,111 @@
+"use strict";
+
+import { dataList, defaultContentItem } from "./data.js";
+
+const elemLogo = document.querySelector(".js-elem-logo");
+const elemMenu = document.querySelector(".js-elem-menu");
+const elemContent = document.querySelector(".js-elem-content");
+
+const createMenuMarkup = (data) => {
+  const menuListItems = data.map((item) => {
+    return `
+                <li class="menu__item" >
+                    <button class="menu__btn" type="button" data-item-id="${item["id"]}">${item["Title"]}</button>
+                </li>
+            `;
+  });
+
+  return `
+            <ul class="menu__list">
+                ${menuListItems}
+            </ul>
+        `;
+};
+
+const changeMinToHm = (min) => {
+  const MIN = Number(min.split(" ")[0]);
+  let h = Math.floor(MIN / 60);
+  let m = Math.abs(MIN % 60);
+
+  return h < 0 ? `${m}m` : `${h}h ${m}m`;
+};
+
+const createСontentMarkup = (item = defaultContentItem) => {
+  const detailsList = item["details"]
+    ? Object.keys(item["details"]).map((key) => {
+        const value =
+          key === "Runtime"
+            ? changeMinToHm(item["details"][key])
+            : item["details"][key];
+
+        return `
+                    <li class="content__info-item">
+                        <span class="content__info-text"><strong>${key}: </strong>${value}</span>
+                    </li>
+                `;
+      })
+    : "";
+
+  return `
+            <header class="content__top">
+                <h1 class="content__title">${item["Title"]}</h1>
+            </header>
+            <div class="content__img-wrap">
+                <figure class="content__img-inner">
+                    <img class="content__img" src="./img/${item["Poster"]}" alt="Poster ${item["Title"]}">
+                </figure>
+            </div>
+            <div class="content__info-wrap">
+                <ul class="content__info-list">
+                    <li class="content__info-item">
+                        <span class="content__info-text content__info-text--plot">${item["Plot"]}</span>
+                    </li>
+                    ${detailsList}
+                </ul>
+            </div>
+        `;
+};
+
+const createInitialMarkup = () => {
+  elemMenu.innerHTML += createMenuMarkup(dataList);
+  elemContent.innerHTML += createСontentMarkup();
+};
+
+createInitialMarkup();
+
+const deleteActiveClass = () =>
+  elemMenu
+    .querySelectorAll(".is-active")
+    .forEach((item) => item.classList.remove("is-active"));
```

Comment:
```suggestion
  elemMenu.querySelector(".js-elem-menu.is-active").classList.remove("is-active");
```
No need to iterate through the whole list, take the needed element with `querySelector`

Reply:
THX for this example, I changed with using optional chaining ?. for a case as class ".is-active" is absent. Done